### PR TITLE
[full-ci] [tests-only] Removed core issue tag, refactored ocis-reva, ocis issue tag

### PR DESCRIPTION
**Pull request by grgprarup**
## Description
Issue tag related to `oC10` are removed, `@issue-ocis-<issue-number>` are refactored to `@issue-<issue-number>`, and for `@issue-ocis-reva-<issue-number>`, tags are removed if the issue is in `ocis-reva` repo, and for those issues moved to `ocis`, tags are refactored to `@issue-<issue-number>`

## Related Issue

- Part of https://github.com/owncloud/ocis/issues/5337

## How Has This Been Tested?

- :robot:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/coreApiAuthWebDav/webDavCOPYAuth.feature [hunk 1]
```diff
@@ -55,7 +55,7 @@ Feature: COPY file/folder
       | /remote.php/dav/spaces/%spaceid%/PARENT/parent.txt |
     Then the HTTP status code of responses on all endpoints should be "401"
 
-  @issue-ocis-reva-14
+
```
Comment (saw-jan) [inline tests/acceptance/features/coreApiAuthWebDav/webDavCOPYAuth.feature:58]: Should we keep reva issue tag?

**Review by SagarGi — COMMENTED** [review]
except above comment LGTM :+1:

File: tests/acceptance/features/coreApiAuthWebDav/webDavCOPYAuth.feature (hunk 1 above)
Comment (grgprarup) [reply tests/acceptance/features/coreApiAuthWebDav/webDavCOPYAuth.feature:58]: The issue was in `ocis-reva` repo and closed there. Do we need this tag anymore? @saw-jan

Comment (saw-jan) [reply tests/acceptance/features/coreApiAuthWebDav/webDavCOPYAuth.feature:58]: No we don't. I thought reva issue

**Review by saw-jan — APPROVED** [review]

**Pull request merged**